components/menu/index.js

```diff
@@ -238,20 +265,28 @@ export function Menu(_context) {
     </section>
     <section class="menu__panel menu__panel--css" id="menu__css" hidden>
       <h4 class="menu__panel-title">
-        <a href="">CSS: Styling language</a>
+        <a href="/en-US/docs/Web/CSS">CSS: Styling language</a>
       </h4>
       <div class="menu__content">
         <div class="menu__content-primary">
           <dl class="menu__content-list">
             <dt>References</dt>
             <dd>
               <ul>
-                <li><a href="">Modules</a></li>
-                <li><a href="">Properties</a></li>
-                <li><a href="">Selectors</a></li>
-                <li><a href="">At-rules</a></li>
-                <li><a href="">Functions</a></li>
-                <li><a href="">See all…</a></li>
+                <li><a href="/en-US/docs/Web/CSS/Modules">Modules [404]</a></li>
+                <li>
+                  <a href="/en-US/docs/Web/CSS/Properties">Properties [404]</a>
+                </li>
+                <li>
+                  <a href="/en-US/docs/Web/CSS/CSS_selectors">Selectors</a>
+                </li>
+                <li>
+                  <a href="/en-US/docs/Web/CSS/CSS_syntax/At-rule">At-rules</a>
+                </li>
+                <li>
+                  <a href="/en-US/docs/Web/CSS/Functions">Functions [404]</a>
+                </li>
+                <li><a href="/en-US/docs/Web/CSS/Reference">See all…</a></li>
```

Comment:
@pepelsbey Are you planning on adding CSS Modules/Properties/Functions overview pages?

cc @Rumyra